### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by AlmightyToast** [THEREALWWEFAN231/Inertia#1063]
**Minecraft Version(s)**
1.12.2
**What is the suggestion**
Auto tool replace.
**Why would this feature be useful**
When I'm mining my nether highway, I would like my tool to be auto replaced so I can go afk. I would also like it to take only 1 tool when it's stacked.
**Final checklist:**

- [ ] I know how to properly use check boxes 
- [ ] This is not a suggestion for baritone or other mod

**Comment by constantinDev** [comment]
i think that may exist, and the name may be autotool

**Comment by feliscatusmeows** [comment]
No, this person wants an autotool that takes 1 of a stacked pickaxe in inventory.

**Comment by AlmightyToast** [comment]
I also want it to replace the tool that i just broke with one i have in my inventory
